modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/legacy/editors.js

```diff
@@ -781,6 +781,7 @@ AUI.add(
 					const inputTpl = Template.get('input');
 					const selectTpl = Template.get('select');
 					const textareaTpl = Template.get('textarea');
+					const checkboxTpl = Template.get('checkbox');
```

Comment:
Hi @ricardocousolr, if we have a customer needing those changes in a previous version, you'll need to send this commit's code in another PR to the specific repository of the legacy version. I think adding this code on master won't change anything in previous versions.